### ISSUE TITLE
docker installation doesn't work well

### ISSUE THREAD

**Issue by masakiaota** [krassowski/jupyterlab-lsp#366]
## Description

The installation Dockerfile will be crush in following line.

`python3 -m pip install --no-cache-dir --no-deps 'jupyter-lsp=0.9.2'`

## Reproduce

1. docker build [Dockerfile](https://jupyterlab-lsp.readthedocs.io/en/latest/Installation.html#Dockerfile)
2. See error
```
ERROR: Invalid requirement: 'jupyter-lsp=0.9.2'
Hint: = is not a valid operator. Did you mean == ?
```

## Expected behavior

docker build correctly

change
`jupyter-lsp=0.9.2`
to
`'jupyter-lsp==0.9.2`

## Context

- Operating System and version: macOS Catalina 10.15.6
- Browser and version: Google Chrome 85.0.4183.83
- JupyterLab version: -

**Comment by krassowski** [comment]
Thank you for the report! I think that will be fixed soon by #365.

**Comment by krassowski** [comment]
I am closing this for now, because the changes in #365 removed the pip install from the docker commands completely, while #367 fixed the other place where we incorrectly had `=` rather than `==` for pip.

Many thanks again for your contributon!
